cmd/webhook/main.go

```diff
@@ -274,15 +269,6 @@ func newConversionController(ctx context.Context, cmw configmap.Watcher, brokers
 					inteventsv1_:       &inteventsv1.Topic{},
 				},
 			},
-			// messaging
-			messagingv1alpha1.Kind("Channel"): {
```

Comment:
Instead of removing it, maybe we keep v1beat1 channel here? I know it is meaningless since now channel only has one version v1bea1 and there is no need to do conversion webhook here. Any suggestions @Harwayne ?

Reply:
ping @Harwayne

Reply:
I think removing it is fine. It will be easy enough to add back when we want to, as there are many other examples to draw upon.